xml/dhcp_vendor_class.xml

```diff
@@ -0,0 +1,215 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<fingerprints matches="dhcp_vendor_class" protocol="dhcp" database_type="service">
+  <!--
+    Fingerprint definitions that are matched against the string values in the
+    dhcp message vi_vendor_class field
+    The vi_vendor_class field can be found in client discover (1), request (3)
+    and inform (8) messages.
+  -->
```

Comment:
If you don't mind, please add wording to indicate that this field is Option 60 as defined in RFC 2132 9.13.